src/api/parser/README.md

```diff
@@ -11,10 +11,6 @@ npm install
 ## Usage
 
 ```
-# normal mode
-npm start
-
-# dev mode with automatic restarts
 npm run dev
```

Comment:
Let's update this so that it mentions `pnpm`, instead of `npm`.

Reply:
When you do, drop the `run` part.